README.md

```diff
@@ -37,4 +37,4 @@ Please note that modifications should follow these coding guidelines:
 Thank you for helping out!
 
 ## Donation
```

Comment:
Let's just remove the entire section, @hd-deman hasn't been active in this organization for years.